soccer/gameplay/tactics/stopped_plays/circle_near_ball.py

```diff
@@ -0,0 +1,147 @@
+import composite_behavior
+import behavior
+import skills.move
+import constants
+import math
+import role_assignment
+import robocup
+import main
+
+
+## Robots position themselves along a portion of the circle centered at the ball
+class CircleNearBall(composite_behavior.CompositeBehavior):
+
+    BackupBallLocation = robocup.Point(0, constants.Field.Length / 4.0)
+
+    def __init__(self):
+        super().__init__(continuous=True)
+
+        self.add_transition(behavior.Behavior.State.start,
+            behavior.Behavior.State.running,
+            lambda: True,
+            'immediately')
+        self.add_transition(behavior.Behavior.State.running,
+            behavior.Behavior.State.completed,
+            lambda: self.all_subbehaviors_completed(),
+            'all robots reach target positions')
+        self.add_transition(behavior.Behavior.State.completed,
+            behavior.Behavior.State.running,
+            lambda: not self.all_subbehaviors_completed(),
+            "robots aren't lined up")
+
+        # Define circle to circle up on
+        radius = constants.Field.CenterRadius + constants.Robot.Radius + 0.01
+
+        perRobot = (2 * constants.Robot.Radius* 1.25) / radius
```

Comment:
Let's remove the positioning calculations from the init method since they duplicate some stuff below.  Maybe change the move_around_circle() method to return 6 Point objects, then you can use it here and in execute_running().